loader/loader.go

```diff
@@ -31,6 +33,7 @@ type ObjectLoader interface {
 	GetRelease(ctx context.Context, cli client.Client, name, namespace string) (*v1alpha1.Release, error)
 	GetReleasePipelineRun(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*tektonv1.PipelineRun, error)
 	GetReleasePlan(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*v1alpha1.ReleasePlan, error)
+	GetReleasePlanAdmissionFromReleasePlan(ctx context.Context, cli client.Client, releasePlan *v1alpha1.ReleasePlan) (*v1alpha1.ReleasePlanAdmission, error)
```

Comment:
Let's rename this to `GetMatchingReleasePlanAdmission`.

Reply:
fixed